package NYql.NYtflow.NProto;


message TQYTSourceMessage {
   optional string Cluster = 1;
   optional string Path = 2;
   optional bytes RowType = 3;
}

message TQYTSinkMessage {
   optional string Cluster = 1;
   optional string Path = 2;
   optional bytes RowType = 3;
}
